Library/Homebrew/formula.rb

```diff
@@ -1810,6 +1819,17 @@ def mirror(val)
       define_method(type) { |val| stable.send(type, val) }
     end
 
+    # @!attribute [w] gpg
```

Comment:
This is currently `rw` as you're accepting a `nil` val as the reader. Just checking this is intentional.